go/worker/compute/executor/committee/node.go

```diff
@@ -1167,22 +1170,48 @@ func (n *Node) proposeBatchLocked(processedBatch *processedBatch) {
 	}()
 	storageCommitLatency.With(n.getMetricLabels()).Observe(time.Since(start).Seconds())
 
-	if err != nil {
+	if storageErr != nil {
+		n.logger.Error("storage failure, submitting failure indicating commitment",
+			"err", storageErr,
+		)
+		proposedResults.SetFailure(commitment.FailureStorageUnavailable)
+	}
+
+	if err := n.signAndSubmitCommitment(proposedResults); err != nil {
+		n.logger.Error("failed to sign and submit the commitment",
+			"commit", proposedResults,
+			"err", err,
+		)
 		n.abortBatchLocked(err)
 		return
 	}
 
-	// Sign the commitment and submit.
-	commit, err := commitment.SignExecutorCommitment(n.commonNode.Identity.NodeSigner, proposedResults)
+	// TODO: Add crash point.
```

Comment:
I don't think we need one as there is one below and nothing interesting happens inbetween.